examples/k8s/eks_sat/README.md

```diff
@@ -0,0 +1,48 @@
+# SPIRE deployment using SAT node attestor on EKS
```

Comment:
I think we're planning on removing all of the k8s examples in lieu of a single example that demonstrates best practices (PSAT attestation + postgres + workload registration, etc). As such, I wouldn't spend too much time developing individual examples. One thing we might do is move any special knowledge or nuance on running this in EKS to the README for the plugin.